en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-remote-h2.md

```diff
@@ -1,429 +1,135 @@
-2.  If you are setting up a separate database to store registry-related data, update the following configurations in the &lt; `PRODUCT_HOME>/repository/conf/registry.xml` file.
-
-    ``` xml
-            <dbConfig name="wso2registry">
-                <dataSource>jdbc/MY_DATASOURCE_NAME</dataSource>
-            </dbConfig>
-    ```
-
-3.  If you are setting up a separate database to store user management data, update the following configurations in the &lt; `PRODUCT_HOME>/repository/conf/user-mgt.xml` file.
-
-    ``` xml
-            <Configuration>
-                <Property name="dataSource">jdbc/MY_DATASOURCE_NAME</Property>
-            </Configuration>
-    ```
+1.  Unzip the WSO2 API Manager pack. Let's call it `<API-M_HOME>`.
 
-### Creating database tables
+1.  Download the [h2 zip file](http://www.h2database.com/html/download.html), and extract it.
 
-To create the database tables, connect to the database that you created earlier and run the following scripts in the H2 shell or web console:
+1.  Copy the JAR file to the `<API-M_HOME>/repository/components/lib/` directory in all the nodes of the cluster.
 
--   To create tables in the registry and user manager database ( `WSO2CARBON_DB` ), use the below script:
-
-    ``` java
-            <PRODUCT_HOME>/dbscripts/h2.sql
-    ```
-
-Follow the steps below to run the script in Web console:
+### Executing db scripts to create tables on remote H2 database
 
 1.  Run the `./h2.sh` command to start the Web console.
-2.  Copy the script text from the SQL file.
-3.  Paste it into the console.
-4.  Click **Run** .
-5.  Restart the server.
-
-        !!! info
-    You can create database tables automatically **when starting the product for the first time** by using the `-Dsetup` parameter as follows:
-
-    -   For Windows: `<PRODUCT_HOME>/bin/wso2server.bat -Dsetup`
-
-    -   For Linux: `<PRODUCT_HOME>/bin/wso2server.sh -Dsetup`
-
-        !!! warning
-        Deprecation of -DSetup
-    When proper Database Administrative (DBA) practices are followed, the systems (except analytics products) are not granted DDL (Data Definition) rights on the schema. Therefore, maintaining the `-DSetup` option is redundant and typically unusable. **As a result, from [January 2018 onwards](https://wso2.com/products/carbon/release-matrix/) WSO2 has deprecated the `-DSetup` option** . Note that the proper practice is for the DBA to run the DDL statements manually so that the DBA can examine and optimize any DDL statement (if necessary) based on the DBA best practices that are in place within the organization.
 
+1.  To create tables in the registry and user manager database (`WSO2_SHARED_DB`) use the script: `<API-M_HOME>/dbscripts/h2.sql`
 
+1.  To create tables in the registry and user manager database (`WSO2_SHARED_DB`) use the script: `<API-M_HOME>/dbscripts/apimgt/h2.sql`
```

Comment:
This has few typos but seems this doc is taken out from our documentation? if so we don't need to fix. @renuka-fernando